projects.yaml

```diff
@@ -117,6 +117,10 @@ projects:
         - *module_if
         - *module_hic_k20dx
         - records/board/rblnano.yaml
+    k20dx_xDot-L151_if:
```

Comment:
This should be lower case and the '-' replaced with an '_', otherwise testing will fail. Currently the code needs to match this regular expression "^([a-z0-9]+)_([a-z0-9_]+)_if$"